src/ui/common/src/components/integrations/connectedIntegrations.tsx

```diff
@@ -46,6 +46,7 @@ export const ConnectedIntegrations: React.FC<ConnectedIntegrationsProps> = ({
       {[...integrations]
         .sort((a, b) => (a.createdAt < b.createdAt ? 1 : -1))
         .map((integration, idx) => {
+          console.log('integrationList render: ', integration);
```

Comment:
Thoughts on architecture:
-Having Integration be the actual integration component and IntegrationCard serve more as a container component seems like a good way to architect this.